s/stringtemplate4/stringtemplate4_ubuntu_16.04.sh

```diff
@@ -0,0 +1,57 @@
+https://github.com/ppc64le/build-scripts/pull/875
+
+# ----------------------------------------------------------------------------
+#
+# Package       : stringtemplate4
+# Version       : 4.3.1
+# Source repo   : https://github.com/antlr/stringtemplate4
+# Tested on     : Ubuntu 20.04.1 LTS (Focal Fossa)
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+export URL=https://github.com/antlr/stringtemplate4
+export BRANCH="$1"
+PKG_NAME=${URL##*/}
+
+if [ -d $PKG_NAME ] ; then
+  rm -rf $PKG_NAME
+fi
+
+#Clean and Install Updates
+sudo apt-get clean
+sudo apt-get update
+sudo apt-get install maven
```

Comment:
Is it required since we are not doing mvn build ? Also isn't ant need to be installed ?

Reply:
Updated the script.